benchopt/plotting/generate_matplotlib.py

```diff
@@ -0,0 +1,61 @@
+import matplotlib.pyplot as plt
+
+
+def get_figures(benchmark, df, output_dir, kinds):
+    "Get the matplotlib figures of the avaible custom plots"
+    figs = []
+    plot_data, _ = benchmark.get_plot_data(df, plotly_style=False)
+    for plot_name in plot_data.keys():
+        if plot_name not in kinds:
+            continue
+        figs.append(get_plot_figure(plot_data[plot_name], df, output_dir))
+    return figs
+
+
+def get_plot_figure(plot_datas, df, output_dir):
+    for key, plot_data in plot_datas.items():
+        if plot_data["type"] == "scatter":
+            return get_plot_scatter(key, plot_data, df, output_dir)
+        else:
+            raise NotImplementedError(
+                f"Plot type {plot_data['type']} "
+                f"not implemented for matplotlib."
+            )
+
+
+def get_plot_scatter(key, plot_data, df, output_dir):
+    df = df.copy()
```

Comment:
Not sure it is needed?
```suggestion
def get_plot_scatter(key, plot_data, output_dir):
```